### ISSUE TITLE
eltype width taken into accounet in display even if it is not shown

### ISSUE THREAD

**Issue by bkamins** [JuliaData/DataFrames.jl#2540]
Example:
```
julia> DataFrame("1" => 1.0, "2" => 2.0)
1×2 DataFrame
 Row │ 1        2       
     │ Float64  Float64 
─────┼──────────────────
   1 │     1.0      2.0

julia> show(DataFrame("1" => 1.0, "2" => 2.0), eltypes=false)
1×2 DataFrame
 Row │ 1        2       
─────┼──────────────────
   1 │     1.0      2.0
```

I think it should be fixed (or is it intended?).

CC @ronisbr

**Comment by ronisbr** [comment]
No, it is not indented. In fact, I have absolutely no clue why this is happening. PrettyTables was not supposed to do that:

```julia
julia> nt = (a = 1:10, b = 1:10)
(a = 1:10, b = 1:10)

julia> pretty_table(nt)
┌───────┬───────┐
│     a │     b │
│ Int64 │ Int64 │
├───────┼───────┤
│     1 │     1 │
│     2 │     2 │
│     3 │     3 │
│     4 │     4 │
│     5 │     5 │
│     6 │     6 │
│     7 │     7 │
│     8 │     8 │
│     9 │     9 │
│    10 │    10 │
└───────┴───────┘

julia> pretty_table(nt, nosubheader = true)
┌────┬────┐
│  a │  b │
├────┼────┤
│  1 │  1 │
│  2 │  2 │
│  3 │  3 │
│  4 │  4 │
│  5 │  5 │
│  6 │  6 │
│  7 │  7 │
│  8 │  8 │
│  9 │  9 │
│ 10 │ 10 │
└────┴────┘
```

**Comment by ronisbr** [comment]
@bkamins sorry! This came from the float pointing alignment. I need to check the size of the header so that the number can be right aligned. I completely forgot that the sometimes the types cam be omitted.

**Comment by bkamins** [comment]
No problem - this is my job to track these things :smile:. Fortunately is is a corner case that is not frequent (and that is why I have released 0.22.0).
